src/main/java/com/Acrobot/ChestShop/ChestShop.java

```diff
@@ -1,10 +1,7 @@
 package com.Acrobot.ChestShop;
 
 import com.Acrobot.Breeze.Configuration.Configuration;
-import com.Acrobot.ChestShop.Commands.Give;
-import com.Acrobot.ChestShop.Commands.ItemInfo;
-import com.Acrobot.ChestShop.Commands.Toggle;
-import com.Acrobot.ChestShop.Commands.Version;
+import com.Acrobot.ChestShop.Commands.*;
```

Comment:
I know that there are already some places that use wildcard imports but I would prefer it if we could avoid introducing new ones, especially if it increases the diff.

Reply:
if you work with IDEA, ctrl+alt+o optimizes the imports and might save you build time